README.md

```diff
@@ -198,6 +199,15 @@ A `TraceApi` object is guaranteed to be returned by both of these calls, even if
 
 A fully detailed overview of the `TraceApi` object is available [here](doc/trace-api.md).
 
+## How does automatic tracing work?
+
```

Comment:
I have addressed the comment in the first paragraph. I am not 100% convinced that reversing the flow of the paragraphs works better – how strongly do you feel about this?